sass/layout/_sidebar-left.scss

```diff
@@ -372,8 +427,11 @@
         width: 100%;
         align-items: center;
 
-        > div {
-            flex: 1 1 auto;
+        > div:not(.SidebarCategory_newLabel) {
```

Comment:
Would it help to make this more specific than saying every div that's not the new category label? I don't know which parts of this are divs that need to get these styles.

Reply:
It's mostly the ones for the menus and the + button for Direct Messages. I can be more specific about this once the PRs are merged together.